test/libsolidity/util/TestFileParser.cpp

```diff
@@ -87,7 +87,11 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 					{
 						tie(call.signature, call.useCallWithoutSignature) = parseFunctionSignature();
 						if (accept(Token::Comma, true))
-							call.value = parseFunctionCallValue();
+						{
+							auto functionValue = parseFunctionCallValue();
+							call.value = functionValue.value * (functionValue.coin == FunctionCallValueCoin::Wei ? 1 : u256(1e18));
```

Comment:
Does `1e18` work on all platforms? Maybe better to use u256 arithmetic.

Reply:
Changed to u256(10)^u256(18)

Reply:
Are you sure this is not xor?